src/menelaus/ensemble.py

```diff
@@ -176,15 +235,10 @@ def update(self, X, y_true=None, y_pred=None):
             #       Need to see why this is happening and where to put e.g. a copy() stmt.
             X_selected = self.column_selectors[det_key](X)
             self.detectors[det_key].update(X=X_selected, y_true=y_true, y_pred=y_pred)
-        self.evaluate()
 
-    def evaluate(self):
-        """
-        Uses evaluator function specified to ensemble, to determine
-        voting result of all detectors. Sets ensemble's own drift
-        state accordingly.
-        """
-        self.drift_state = self.evaluator(self.detectors.values())
+        # TODO - reset ensemble itself and its detectors, right now never reset
```

Comment:
Should be taken care of by the call `StreamingEnsemble.reset -> StreamingDetector.reset`.